gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.jvmargs=-Xmx256m
```

Comment:
I run this code on a low-memory VM (1GB ram) and this is helpful for building there. I can remove it if anyone objects but it seems helpful in general.